packages/Telephony/Actions/CallTransfer.cs

```diff
@@ -55,12 +56,24 @@ public CallTransfer([CallerFilePath] string sourceFilePath = "", [CallerLineNumb
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         public async override Task<DialogTurnResult> BeginDialogAsync(DialogContext dc, object options = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            var phoneNumber = this.PhoneNumber?.GetValue(dc.State);
+            if (dc.Context.Activity.ChannelId == Channels.Telephony)
+            {
+                var error = "Phone number is empty, please provide number in E.164 format";
+                var regexFormat = "^\\+\\d{1,3}\\s\\d{1,14}(\\s\\d{1,13})?";
```

Comment:
I cannot tell if this regex meets E.164  standard :) Likely we might miss something: https://www.itu.int/rec/T-REC-E.164-201011-I/en

Reply:
This regex is not right, the code below returns `false` when I just tested it in vacuum using your test phone number

```csharp
	public static void Main()
	{
		var regexFormat = "^\\+\\d{1,3}\\s\\d{1,14}(\\s\\d{1,13})?";
		Console.WriteLine(Regex.IsMatch("+1555443443", regexFormat));
		
	}
```

Regardless, let's discuss on Tuesday when we go through the spec since I'm not sure that regex validation is a good thing to do here. And if we do want it, we can discuss the trade offs of different approaches together in a call which is more efficient.

Reply:
thanks, this is the issue with not using standard validation here, removing this validation as we agreed upon